products/appengine/terraform.yaml

```diff
@@ -93,6 +93,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  ApplicationUrlDispatchRules: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{project}}"
+    import_format: ["{{project}}"]
+    mutex: "apps/{{project}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      test_check_destroy: templates/terraform/custom_check_destroy/appengine.go.erb
```

Comment:
Can you remove the old `custom_check_destroy` file and use the same one for both appengine resources?

Reply:
Its done